nuget_test.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"testing"
+
+	"github.com/jfrog/jfrog-cli-go/inttestutils"
+	"github.com/jfrog/jfrog-cli-go/utils/cliutils"
+	"github.com/jfrog/jfrog-cli-go/utils/tests"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+)
+
+func initNugetTest(t *testing.T) {
+	if !*tests.TestNuget {
+		t.Skip("Skipping NuGet test. To run Nuget test add the '-test.nuget=true' option.")
+	}
+
+	if !cliutils.IsWindows() {
+		t.Skip("Skipping nuget tests, since this is not a Windows machine.")
+	}
+
+	// This is due to Artifactory bug, we cant create remote repository with REST API.
+	if !isRepoExist(tests.NugetRemoteRepo) {
+		t.Error("Create nuget remote repository:", tests.NugetRemoteRepo, "in order to run nuget tests")
+		t.FailNow()
+	}
+}
+
+func TestNugetResolve(t *testing.T) {
+	initNugetTest(t)
```

Comment:
It looks like the *initNugetTest* function is not invoked from where the other init functions are invoked.